### ISSUE TITLE
Cross entropy sequence labelling scores are computed for each element not for all sequence as in CRF

### ISSUE THREAD

**Issue by vladosdudos** [ispras-texterra/derek#44]
In cross-entropy loss `scores` are [batch_size, seq_len, out_size] tensor, in crf are [batch_size]. This could lead to bugs in further code.

https://github.com/ispras-texterra/derek/blob/92a1c90007d66121fadd0e4564942798eeffd7f3/derek/common/nn/tf_losses.py#L27-L48
